trunk/bin/lscloop.py

```diff
@@ -138,10 +139,7 @@ def multi_run_cosmic(args):
                            str(ll['mag'][i]), ll['abscat'][i])
             print '\n###  total number = ' + str(len(ll['filename']))
             if args.standard:
-                if args.standard == 'all':
-                    mm = lsc.myloopdef.get_standards(args.epoch, args.name, filters)
-                else:
-                    mm = lsc.myloopdef.get_list(args.epoch, args.telescope, filters, _instrument=args.instrument, _name=args.standard)
+                mm = lsc.myloopdef.get_standards(args.epoch, args.name, filters, args.standard, args.match_by_site)
```

Comment:
How did we get rid of the if statement here? Did we refactor that logic into get_standards?

Reply:
Yes, and that is a big improvement because now the matching is handled the same way whether it's one standard or many.